shards/egui/src/widgets/image.rs

```diff
@@ -145,9 +154,17 @@ impl Image {
     }
   }
 
-  fn activate_texture(&mut self, _context: &Context, input: &Var) -> Result<Var, &str> {
+  fn activate_texture(&mut self, context: &Context, input: &Var) -> Result<Var, &str> {
+    // support rendering multiple images from same shard
+    let current_step = shards::core::step_count(context);
+    if self.step_counter != current_step {
+      self.step_counter = current_step;
+      self.step_textures.clear();
+    }
+
     if let Some(ui) = util::get_current_parent_opt(self.parents.get())? {
-      let (texture_id, texture_size) = image_util::get_egui_texture_from_gfx(input)?;
+      let (texture_id, texture_size) =
```

Comment:
Isn't it easier to just store it on the UI context? 
we have a custom rust struct for that already

Reply:
Yeah, I suppose, I will check

Reply:
But for example in this Shard it's not available, is that normal btw? I guess `parents` include it?